.github/workflows/build.yml

```diff
@@ -1,9 +1,8 @@
 name: Build
 
 on:
+  pull_request:
   push:
-    branches:
```

Comment:
I would leave this in otherwise you get a build whenever you push a new branch and duplicates on PR as `pull_request` will re-run when you push to a PR

Reply:
Ty! Noticed that last night